### ISSUE TITLE
Recommended way for adding Errata to articles?

### ISSUE THREAD

**Issue by sdaau** [zenodo/zenodo#1585]
Hi all,

Is there a recommended way to add Errata for journal/conference articles on Zenodo?

For instance, I can see that when I'm logged in as a curator of a community, for a paper belonging to the community ( say, https://zenodo.org/record/849141 ), there is a "New version" button, with the note:

> You should only create a new version if you want to update the files of this upload. Keep in mind that new versions get a new DOI.

However. for the paper link above, there is also this a but further down on the page:

> Version 1            10.5281/zenodo.849141 | Oct 20, 2004
> -- | --
> Cite all versions? You can cite all versions by using the DOI 10.5281/zenodo.849140. This DOI represents all versions, and will always resolve to the latest one. Read more.

![zenodo_paper-fullpage](https://user-images.githubusercontent.com/26737749/44456974-40ac9e00-a602-11e8-8ec9-e60a125cdb01.png)

Basically, I would like the Errata to be associated to the original paper, as in a new version case - however, some Erratas contain *just* the corrections to the original paper, so in that sense, they are not really "new versions" of the original paper.

Would it be acceptable to add Erratas of papers as new versions of the original paper - and if now, is there another recommended facility to add erratas for papers in Zenodo, so they (their DOI, that is) is associated to the original paper?

**Comment by moble** [comment]
The errata shouldn't be included *alone* in the new version; instead it should be added as a file *alongside* the original paper.  In this case, you should probably note that the main paper file has some errata contained in the other file.

Alternatively, if you can generate it (which I'm guessing you can because that looks like a basic LaTeX document), you could just upload the corrected paper itself as the only file in the new version, and not worry about including the separate "Errata" document as a file.  In this case, you should definitely add an explanation to the description field that this is the corrected version of the paper, and possibly a brief list mentioning just what the corrections are.

In either case, if the Errata has been published elsewhere, you can add its identifier to the "Related Identifiers" field with "is a supplement to this upload".

**Comment by sdaau** [comment]
Many thanks for the response, @moble :

> The errata shouldn't be included _alone_ in the new version; instead it should be added as a file _alongside_ the original paper. In this case, you should probably note that the main paper file has some errata contained in the other file.

Thanks, this is the kind of specification I was looking for.

> Alternatively, if you can generate it (which I'm guessing you can because that looks like a basic LaTeX document), you could just upload the corrected paper itself as the only file in the new version, and not worry about including the separate "Errata" document as a file. In this case, you should definitely add an explanation to the description field that this is the corrected version of the paper, and possibly a brief list mentioning just what the corrections are.

Indeed, it is a basic LaTeX document in the case of our community (SMC - Sound and Music Computing conference), - but it can be difficult to get a hold of authors, and have them generate an additional corrected version (or anything beyond what they've already submitted), especially if years have passed by.

So, I just tried hitting Edit on the [paper](https://zenodo.org/record/849141) from the OP screenshot (it's not that paper that needs the errata, I just used it as an example) - as a try, to see if I could add another file after a record has been published, and I get:

![zenodo-edit](https://user-images.githubusercontent.com/26737749/44543004-a7b77900-a70e-11e8-8588-db46575b91bf.png)

>> Note: File addition, removal or modification are not allowed after an upload  has been published. This is because a Digital Object Identifier (DOI) is  registered with  DataCite for each upload. If you've made a mistake please contact us.
>> \-\-  
>> If you have an updated version of your files, you can create a new record version, with a completely 

So, in this case, would be following suggestion be acceptable?:

* Click on "New version"
* In the new version, upload the old paper PDF, *and* the errata PDF
* Add a note in the new version that it contains the original PDF, along with errata

> In either case, if the Errata has been published elsewhere, you can add its identifier to the "Related Identifiers" field with "is a supplement to this upload".

Ah, great - so it means I could have a standalone record for the errata, too! So, in case my previous suggestion is not acceptable, I can definitely use this...

**Comment by sdaau** [comment]
Many thanks @moble , I think my questions are answered now. I'll leave this issue as open, so the project administrators can decide whether some of this discussion for errata needs to end up in the official documentation for Zenod or not.
